org.freedesktop.LinuxAudio.BaseExtension.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id" : "org.freedesktop.LinuxAudio.BaseExtension",
+  "branch": "20.04",
+  "runtime" : "org.freedesktop.Platform",
+  "sdk" : "org.freedesktop.Sdk",
+  "runtime-version" : "19.08",
+  "finish-args": [
```

Comment:
I don't think `finish-args` are inherited by applications using baseapps, unless you put it here for reference.

Reply:
Apps don't use the base app at all. This is only to build plugins.

Otherwise they are here for reference but maybe I should just put that verbatim in the README that act as a documentation for now.

Reply:
Yes, it was a mental leap, I meant plugins. +1 for putting it into README.